svcs/guacamole/intercept.go

```diff
@@ -309,10 +211,10 @@ func (gtl *guacTokenLoginEndpoint) Intercept(next http.Handler) http.Handler {
 			}
 			u, err := gtl.users.GetUserForTeam(t.Id)
 			if err != nil {
-				log.Warn().
```

Comment:
Why is this commented out?

Reply:
:( Actually, it is  commented out because waiting page has interval refresh rate of 10 seconds, after 10 seconds it refresh the page, but if there is no lab to assign the log will be printed out into daemon service log, until they assigned to a lab, same log statement created a lot of times, maybe 30 or 40 lines which increased redundancy on server side. Therefore, it is removed, but I accept that it is definitely right solution to do. Maybe having long refresh interval time or no auto refresh on waiting page might be more acceptable approach. However, again it is great to have feedbacks :)